.github/workflows/update-crate-versions.yml

```diff
@@ -0,0 +1,43 @@
+name: Update crates versions
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  schedule:
+    # * is a special character in YAML, needs quoting
+    - cron:  '0 0 * * 0'
+  workflow_dispatch:
+
+jobs:
+  updateversions:
+    runs-on: ubuntu-latest
+    container:
+      image: rust:latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Retrieve newest versions of dependencies
+        run: |
+          echo 'cargo_toml<<EOF' >> $GITHUB_ENV
+          ./bin/update-crates-versions.sh | sed 's/"/\\"/g' >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: Update Cargo.toml
+        run:
+          echo "${{env.cargo_toml}}" > local-registry/Cargo.toml
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
```

Comment:
Same comment as above. @exercism/github-actions what do you think about this action?

Note that you can also use the [GitHub CLI](https://github.com/cli/cli), which comes pre-installed.

Reply:
Oh, all right, I didn't know that the cli tools came pre-installed, I can create a PR this way if it's preferable.

Reply:
I think it might be slightly less susceptible to misuse (or errors)

Reply:
Wondering if that would actually work since I'm specifying container to be `rust:latest`...

Reply:
It won't. That said, it might be worth changing this workflow to be like https://github.com/exercism/rust/blob/main/.github/workflows/tests.yml#L113, which is the tests workflow for the rust repo. In it, the base image is ubuntu, but the `action-rs` action is used to install rust.

But let's wait for @exercism/github-actions to see what they think.

Reply:
All right ! Ignore my latest changes then.

Reply:
I think the `gh` way is simpler from a maintenance PoV but I don't see anything wrong with using that action (as long as it's pinned to a trusted SHA)

Reply:
what about now?

Reply:
Hello, it's been a few months and I'd like to know why you stopped reviewing, it was nice contributing to exercism :)

Reply:
I'm really not the right person to review this, but at a guess, the people who'd been working with you on this PR got distracted and then just forgot.

Reply:
I'm so sorry @dhovart! I guess @SaschaMann and I just forgot to review this. Our apologies.

Reply:
Ah, no worries at all :)